t/146-shdict-decr-init.t

```diff
@@ -0,0 +1,248 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+use lib 'lib';
```

Comment:
Whups, this was copied from test 144.There are a few new tests that have this, likely worth a separate PR to clean up these tests for style.